WireFoundation/Sources/WireFoundation/ReadOnlyCurrentValueSubject.swift

```diff
@@ -16,6 +16,25 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
+public import Combine
 
-// empty for now
+/// Helper subject to be able to pass object from which you can get current value and publisher
+/// but not able to send values as to regular subject (safety)
+public final class ReadOnlyCurrentValueSubject<Output> {
```

Comment:
As discussed, it would be great to make `ReadOnlyCurrentValueSubject` conform to `Publisher`, and possibly name it something like `CurrentValuePublisher`.